### PR TITLE
tidied up SEO_Title meta display to just show the SEO_Title

### PR DESCRIPTION
**Pull request by Rambowalsh**
### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1811
- [x] Jira ticket has the correct status.
- [ ] [Changelog](CHANGELOG.md) entry added. - it's all part of the original entry

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: domestic/templates/domestic/landing_page.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
   <script src="{% static 'js/home.js' %}"></script>
 {% endblock %}
 
-{% block head_title %}{% firstof page.seo_title page.title %} - great.gov.uk{% endblock %}
+{% block head_title %}{% firstof page.seo_title page.title%}{% endblock %}
```
Comment (webbyfox) [inline domestic/templates/domestic/landing_page.html:18]:
```suggestion
{% block head_title %}{% firstof page.seo_title page.title %}{% endblock %}
```

**Review by webbyfox — CHANGES_REQUESTED** [review]
Getting rid of the domain, Is there a ticket for this? Change log update, please.

**Review by webbyfox — APPROVED** [review]
LGTM

**Comment by Rambowalsh** [comment]
Removed domain from meta seo title

**Pull request merged**